R/init.R

```diff
@@ -117,11 +117,27 @@ init_last <- function() {
     }
 
     if (!is.null(fileCon)) {
+      rSettings <- c(
+        "plot.useHttpgd",
+        "workspaceViewer.showObjectSize",
+        "session.emulateRStudioAPI",
+        "session.levelOfObjectDetail",
+        "session.watchGlobalEnvironment",
+        "session.viewers.Plot Column",
```

Comment:
Could we eliminate the space here? Looks a bit odd.

Reply:
I agree, the issue I was having is that array items don't use the same naming scheme as other configuration names, so `plotColumn` does not display as `Plot Column` in an array.

Reply:
Maybe the viewers could be instead grouped under `session.viewers.viewColumns.x`, which also leaves space for more configurations in the future

Reply:
I agree. Looks good to me if we just put them as ordinary settings like others.

Reply:
do you think camel case is better for the two-word options, e.g.

![image](https://user-images.githubusercontent.com/60372411/128799927-fc3d8a06-6d93-403c-8bdf-4e9728bcfc76.png)

Or perhaps:

![image](https://user-images.githubusercontent.com/60372411/128799972-416ff4b9-8575-43fd-8556-693c0ca2b721.png)

These would be seen in R as `session.viewers.viewColumn.camelCase` or `session.viewers.viewColumn.snake_case`

Reply:
`session.viewers.viewColumn.camelCase` is more consistent with other settings.